lib/ProductOpener/Display.pm

```diff
@@ -78,6 +78,7 @@ BEGIN
 
 					$original_subdomain
 					$subdomain
+					$format_subdomain
```

Comment:
I think, `$formatted_subdomain` might be a better name, because the formatting has already been done.

Reply:
Agreed.